gensim/models/tfidfmodel.py

```diff
@@ -273,21 +277,34 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
                 * `c` - cosine.
 
             For more information visit [1]_.
-
+        pivot_norm : bool, optional
+            If pivot_norm is True, then pivoted document length normalization will be applied.
+        slope : float, optional
+            It is the parameter required by pivoted document length normalization which determines the slope to which
+            the `old normalization` can be tilted.
+        pivot : float, optional
+            Pivot is the point before which we consider a document to be short and after which the document is
+            considered long. It can be found by plotting the retrieval and relevence curves of a set of documents using
```

Comment:
also, you can add latex formula here using http://www.sphinx-doc.org/en/master/ext/math.html